scene/gui/tab_bar.cpp

```diff
@@ -1676,7 +1676,7 @@ bool TabBar::_get(const StringName &p_name, Variant &r_ret) const {
 	Vector<String> components = String(p_name).split("/", true, 2);
 	if (components.size() >= 2 && components[0].begins_with("tab_") && components[0].trim_prefix("tab_").is_valid_int()) {
 		int tab_index = components[0].trim_prefix("tab_").to_int();
-		String property = components[1];
+		const String &property = components[1];
```

Comment:
Change the below use of `components[1]` to `property` here

Reply:
Handled in #85071.